qui/tray/devices.py

```diff
@@ -139,12 +145,25 @@ def __init__(self, device, *args, **kwargs):
         self.add(self.hbox)
 
 
+class DevclassHeaderMenuItem(Gtk.MenuItem):
+    """ MenuItem with a header, non-interactive """
+
+    def __init__(self, devclass, *args, **kwargs):
+        super(DevclassHeaderMenuItem, self).__init__(*args, **kwargs)
+
+        label = Gtk.Label(xalign=0)
+        label.set_markup("<b>{}</b>".format(
+            DEV_TYPE_NAMES.get(devclass, "Other Devices")))
+
+        self.add(label)
```

Comment:
I think you need to make the entry non-clickable.

Reply:
That would influence the display, and it seems nobody notices if it is clickable-but-useless (compare qui-domains)